cmd/websocketsource/main.go

```diff
@@ -28,19 +28,21 @@ import (
 	"github.com/knative/pkg/cloudevents"
 )
 
+const (
+	eventType = "dev.knative.event.websocket"
```

Comment:
I think these should be: dev.knative.eventing
to match:
pkg/apis/eventing/register/register.go
where our GroupName is eventing.knative.dev

so s/event/eventing/ to make sure they match.